### DISCUSSION TITLE
Extend with JS, increment instead of replace

### DISCUSSION THREAD

**Discussion by 0x6775737461766f** [pocketbase/pocketbase#3935] [Q&A] [answered]
So basically, i have a field "members" on my clubs records, it is a multiple relations field linking to users, but when i do this:
`const record = $app.dao().findRecordById("clubs", charge.metadata.id_club);
                if (!record) {
                    throw new NotFoundError();
                }
                record.set("members", charge.client_reference_id);
                $app.dao().saveRecord(record);
                return c.json(201, { status: true, message: 'Membro adicionado com sucesso!' });`
             
it replaces the current user for the one i send on the api request, instead, how can i "stack" them, i.g add multiple members instead?

**Comment by ganigeorgiev** [comment, marked as answer]
You have to retrieve the existing value, append or reorder with the new one and then set it back. Something like:

```js
const members = record.get("members");

members.push(charge.client_reference_id)

record.set("members", members);
```

**Reply by 0x6775737461766f** [reply]
Thank you for the quick reply
